GitUI/CommandsDialogs/SettingsDialog/Pages/FormChooseTranslation.cs

```diff
@@ -67,7 +67,13 @@ private void FormChooseTranslation_FormClosing(object sender, FormClosingEventAr
 
         private void lvTranslations_ItemActivate(object sender, EventArgs e)
         {
-            AppSettings.Translation = ((ListView)sender).SelectedItems[0].Tag.ToString();
+            var selectedItems = ((ListView)sender).SelectedItems;
+            if (selectedItems.Count > 0)
+            {
+                AppSettings.Translation = selectedItems[0].Tag.ToString();
+            }
+            //// else see fallback in FormChooseTranslation_FormClosing
```

Comment:
nitpick: This is a comment rather than commented code, I suggest to use //
(apreciate the comment though)

Reply:
Then StyleCop would enforce an empty line above the comment...

Reply:
I would have put the comment after line 72